### PR TITLE
chore: squash Kommander release notes for 1.1.0 GA

### PR DESCRIPTION
**Pull request by samvantran**
## Description of changes being made

This commit combines all release notes for betas and rcs into
a comprehensive diff against the 1.0 version. Also tried to highlight
the new features.

### CODE REVIEW

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 1]
```diff
@@ -50,369 +50,109 @@ Template:
...
-### Kommander 1.1.0 RC 7 - July 16th 2020
+#### New Features
+
+##### Centralized Cost Monitoring
+
+Kubecost, running on Kommander, provides centralized cost monitoring for all managed clusters. This feature, installed by default in every Kommander cluster, provides a centralized view of Kubernetes resources used on all managed clusters. For more information go to [Centralized Cost Monitoring](https://docs.d2iq.com/ksphere/kommander/1.1.0/centralized-cost-monitoring/)
+
+##### D2-ify Managed Clusters
+
+When attaching non-Konvoy clusters, such as Amazon EKS, Azure AKS, Google GKE, and On-Premises Kubernetes clusters, Kommander will federate a subset of standard Konvoy Addons to enable SSO, AuthN/Z, observability, and cost monitoring. This includes addons: Prometheus Operator, Alert Manager, Karma, Elasticsearch, Kibana, and Fluentbit.
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1/release-notes/index.md:68]: Couldn't find a doc describing the addons we federate

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 2]
```diff
@@ -50,369 +50,109 @@ Template:
...
+
+##### Improved RBAC
+
+Kommander has revamped Access Controls for users at the global, workspace, and project levels allowing for greater flexibility and security when assigning roles. See [Granting Access to Kubernetes Resources in Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/tutorials/configure-rbac/)
+
+##### LDAP Support
+
+Kommander now supports configuring an external LDAP directory for authenticating users. See [Connect and configure your LDAP directory to Konvoy and Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/ldap/setup-ldap/)
```
Comment (samvantran) [inline pages/ksphere/kommander/1.1/release-notes/index.md:76]: Not 100% sure if this was supported before but saw new docs for it

Comment (juliangieseke) [reply pages/ksphere/kommander/1.1/release-notes/index.md:76]: thats old. but got some improvements IIRC. But SAML was added AFAIK.

File: pages/ksphere/kommander/1.1/release-notes/index.md (hunk 1 above)
Comment (cmastr) [reply pages/ksphere/kommander/1.1/release-notes/index.md:68]: If we think this needs additional information, we can open a JIRA. We have to remember that release notes may not stick around forever, so making sure we talk about this stuff outside of release notes is usually necessary.

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 3]
```diff
@@ -49,370 +49,117 @@ Template:
...
+
+Kubecost, running on Kommander, provides centralized cost monitoring for all managed clusters. This feature, installed by default in every Kommander cluster, provides a centralized view of Kubernetes resources used on all managed clusters. For more information go to [Centralized Cost Monitoring](https://docs.d2iq.com/ksphere/kommander/1.1.0/centralized-cost-monitoring/)
+
+##### D2-ify Managed Clusters
+
+When attaching non-Konvoy clusters, such as Amazon EKS, Azure AKS, Google GKE, and On-Premises Kubernetes clusters, Kommander will federate a subset of standard Konvoy Addons to enable SSO, AuthN/Z, observability, and cost monitoring. This includes addons: Prometheus Operator, Alert Manager, Karma, Elasticsearch, Kibana, and Fluentbit.
+
+##### Improved RBAC
+
+Kommander has revamped Access Controls for users at the global, workspace, and project levels allowing for greater flexibility and security when assigning roles. See [Granting Access to Kubernetes Resources in Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/tutorials/configure-rbac/)
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:72]:
```suggestion
Kommander has enhanced Access Controls for users at the global, workspace, and project levels allowing for greater flexibility and security when assigning roles. See [Granting Access to Kubernetes Resources in Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/tutorials/configure-rbac/)
```

File: pages/ksphere/kommander/1.1/release-notes/index.md (hunk 2 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.1/release-notes/index.md:76]: I updated that topic based on Docathon input. It a long topic but hopefully it covers everything.

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 4]
```diff
@@ -49,370 +49,117 @@ Template:
...
+
+##### LDAP Support
+
+Kommander now supports configuring an external LDAP directory for authenticating users. See [Connect and configure your LDAP directory to Konvoy and Kommander](https://docs.d2iq.com/ksphere/kommander/1.1.0/ldap/setup-ldap/)
+
+#### Improvements
+
+Beyond new features, here are the improvements you can expect when upgrading from Kommander 1.0:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:80]:
```suggestion
Beyond new features, here are the improvements made in Kommander 1.1:
```

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 5]
```diff
@@ -49,370 +49,117 @@ Template:
...
+* Display users Username in the UI
+* Enabled License validation
+* Generated labels are now hidden in cluster overview pages
+* Hid Kubeconfig download link for clusters where that config might not be available
+* Improved Grafana dashboard names for addons
+* Improved auth token transport for managed clusters
+* Improved context handling for multiple contexts when attaching cluster
+* Improved display of resource allocations in the UI
+* Improved attach Cluster Flow in UI
+* Improved K8s Version Selector and Support for managed clusters created via UI
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:107]:
```suggestion
* Improved K8s Version Selector and Support for managed clusters created through the UI
```

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 6]
```diff
@@ -49,370 +49,117 @@ Template:
...
+* Improved AWS tags to include cluster name.
+* Improved Error messaging when trying to delete roles or groups that are used by policies
+* Improved performance for querying available versions in cluster create form.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:110]:
```suggestion
* Improved performance for querying available versions in cluster create form
```

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 7]
```diff
@@ -49,370 +49,117 @@ Template:
...
+* Improved Cluster Status Visualisation in the UI
+* Removed suffixes from federated ConfigMaps and Secrets
+* Removed namespace suffix from projects and platform service names
+* Removed old, unsupported versions from version selector in create cluster form.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:114]:
```suggestion
* Removed old, unsupported versions from version selector in create cluster form
```

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 8]
```diff
@@ -49,370 +49,117 @@ Template:
...
 
-- Fixed kubeconfig download for certain clusters/users
-- Fixed uninstall from Kommander addon
-- Lots of smaller UX Bugs and Improvements
+* Allow deleting clusters retry after failing. For example when there are permission issues.
+* Disabled "View Logs" Link in UI for managed clusters not running Kibana
+* Federate karma-proxy only to Konvoy clusters
+* Fixed missing Grafana home dashboard
+* Fixed cluster deletion on detail page not working
+* Fixed a bug where Kommander addon wasnt successfully deploying on Azure
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:128]:
```suggestion
* Fixed a bug where Kommander addon was not successfully deploying on Azure
```

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 9]
```diff
@@ -49,370 +49,117 @@ Template:
...
+* Fixed Kommander to not show unofficially supported versions of Kubernetes by default
+* Fixed possible data collision bug related to clusters
+* Fixed a possible crash-loop situation for kubeaddons when cert-manager isnt ready yet
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:131]:
```suggestion
* Fixed a possible crash-loop situation for kubeaddons when cert-manager is not ready yet
```

File: pages/ksphere/kommander/1.1/release-notes/index.md [hunk 10]
```diff
@@ -49,370 +49,117 @@ Template:
...
+* Fixed and improved LDAP Identity provider handling
+* Fixed display of "nothing to report" situations vs actual errors
+* Fixed bugs related to access control
+* Fixed naming of roles in projects
+* Fixed an issue where projects were not created due to a bug in project name suffix handling
+* Fixed version selector when creating clusters via UI
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/release-notes/index.md:137]:
```suggestion
* Fixed version selector when creating clusters through the UI
```

File: pages/ksphere/kommander/1.1/release-notes/index.md (hunk 1 above)
Comment (samvantran) [reply pages/ksphere/kommander/1.1/release-notes/index.md:75]: yup i've asked @drewvanstone's team to write up a doc. We can link to it after

File: pages/ksphere/kommander/1.1/release-notes/index.md (hunk 2 above)
Comment (samvantran) [reply pages/ksphere/kommander/1.1/release-notes/index.md:83]: hm... there is literally not a single mention of SAML in all the kommander docs. And from this doc https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/operations/identity-providers/

```
Currently, Kommander supports Github, LDAP, and any standard OIDC provider such as Google.
```

Lucy you got some 'splaining to do

Anyway if LDAP is old I'll take it out from the new features section

**Comment by cmastr** [comment]
I modified the structure a bit more to be in line with what we do in DC/OS.

File: pages/ksphere/kommander/1.1/release-notes/index.md (hunk 2 above)
Comment (juliangieseke) [reply pages/ksphere/kommander/1.1/release-notes/index.md]: yeah… was referring to https://jira.d2iq.com/browse/D2IQ-65025, but it didnt made it into the UI 🙈  we got a JIRA to add it and will update docs once thats done.

Comment (juliangieseke) [reply pages/ksphere/kommander/1.1/release-notes/index.md]: so yeah, removing it for this release is probably correct.

**Pull request merged**